iam_assumerole/main.tf

```diff
@@ -45,6 +45,11 @@ variable "role_name" {
   description = "Name of the IAM role."
 }
 
+variable "permissions_boundary_policy_arn" {
+  description = "ARN of the permission boundary policy"
+  default     = ""
```

Comment:
Similarly -- if this ARN is required, this value _cannot_ be left blank, so a default value should not be provided:
```suggestion

Reply:
Tested and removed default = "". The calling module which sends the arn of the pb policy ensures that a value is sent. That PR is in another repo.